prow/plugins/trigger/pr.go

```diff
@@ -128,6 +136,10 @@ func trustedPullRequest(ghc githubClient, pr github.PullRequest) (bool, error) {
 }
 
 func buildAll(c client, pr github.PullRequest) error {
+
+	if err := c.GitHubClient.RemoveLabel(pr.Base.Repo.Owner.Login, pr.Base.Repo.Name, pr.Number, needsOkToTest); err != nil {
+		c.Logger.Errorf("Failed at removing %s label", needsOkToTest)
```

Comment:
Don't log it, propagate it up! In this case it's fine to break here. If we hit an error in this API call then it means something is very wrong, and it's fine to not start the jobs in this case.